lnbits/core/static/js/wallet.js

```diff
@@ -119,8 +119,8 @@ new Vue({
         paymentHash: null,
         minMax: [0, 2100000000000000],
         lnurl: null,
-        units: ['sat'],
-        unit: 'sat',
+        units: ['USD'],
+        unit: 'USD',
```

Comment:
I dont really get these ones? The `unit` is always set to `sat` initially when opening the receive dialog and the units list shouldnt contain USD by default because it might not be included in the list of allowed currencies.

Reply:
Good point, I was going off wallets like WoS, which defaults to a fiat

Reply:
But feel free to revert